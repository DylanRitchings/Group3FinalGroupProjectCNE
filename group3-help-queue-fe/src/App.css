.screenDiv {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  text-align: center;
}

.queue {
  width: 44%;
  height: 70%;
  position: absolute;
  top: 10%;
  left: 5%;
  border: 2px solid black;
}

.completed {
  width: 44%;
  height: 70%;
  position: absolute;
  top: 10%;
  right: 5%;
  border: 2px solid black;
}