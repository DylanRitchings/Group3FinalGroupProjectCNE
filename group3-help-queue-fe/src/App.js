import './App.css';
import CompletedTickets from './completedTickets';
import QueuedTickets from './queuedTickets';
import WelcomeUser from './welcomeUser';
import { useState } from 'react';

function App() {

  const [tickets, setTickets] = useState([
    {

        id:1,
        usersName: "Luke Foster",
        time: "17:14",
        ticketDescription: "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Error perferendis porro consequuntur illum omnis quas ex incidunt eius eos sed magni earum iusto maiores laboriosam numquam maxime, quidem nobis sint.",
        ticketTitle: "Docker Problem"

    },
    {
        id:2,
        usersName: "Jack Smith",
        time: "17:36",
        ticketDescription: "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Error perferendis porro consequuntur illum omnis quas ex incidunt eius eos sed magni earum iusto maiores laboriosam numquam maxime, quidem nobis sint.",
        ticketTitle: "Terraform Problem"
    },
    {
        id:3,
        usersName: "Ben Wellens",
        time: "11:25",
        ticketDescription: "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Error perferendis porro consequuntur illum omnis quas ex incidunt eius eos sed magni earum iusto maiores laboriosam numquam maxime, quidem nobis sint.",
        ticketTitle: "Login Problem"
    },
    {
        id:4,
        usersName: "Sam Matterson",
        time: "14:52",
        ticketDescription: "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Error perferendis porro consequuntur illum omnis quas ex incidunt eius eos sed magni earum iusto maiores laboriosam numquam maxime, quidem nobis sint.",
        ticketTitle: "500 Server error"
    }
  ])

  return (
    <div className="screenDiv">
      <WelcomeUser/>
    <div className="queue">
      <QueuedTickets tickets={tickets} setTickets={setTickets}/>
    </div>
    <div className="completed">
      <CompletedTickets/>
    </div>
    </div>
  );
}

export default App;
