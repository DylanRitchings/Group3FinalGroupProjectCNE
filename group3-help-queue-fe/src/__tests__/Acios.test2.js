import React from 'react';
import renderer from 'react-test-renderer';
import deleteTicket from  '../cardStructure';
import App from '../App';

it('renders correctly', () => {
    const id = 2;
    const tree = renderer.create(<deleteTicket id={id} />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
});