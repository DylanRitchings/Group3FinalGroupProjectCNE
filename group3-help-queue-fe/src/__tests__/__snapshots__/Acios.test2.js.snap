// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<deleteTicket
  id={2}
/>
`;

exports[`renders correctly 2`] = `
<p>
  Please wait.... we are getting your request
</p>
`;
